File: Easy/circle_or_square.py
# https://edabit.com/challenge/4me7LifXBwj5rhL4n
from math import pi,sqrt
def circle_or_square(rad, area):
    return (2*pi)*rad>sqrt(area)*4
    # return 2*pi*(rad*.5)>(area*.5)*4

print(circle_or_square(16, 625), True)
print(circle_or_square(8, 144), True)
print(circle_or_square(15, 400), True)
print(circle_or_square(5, 100), False)
print(circle_or_square(18, 900), False)
print(circle_or_square(1, 4), False)

File: javascript/Very_Easy/circle_or_square.js
// https://edabit.com/challenge/D8DDFMFK8RaWWmcGY


function circle_or_square(rad, area){
    return 2*Math.PI*rad>Math.sqrt(area)*4
}


console.log((circle_or_square(16, 625), true));
